GenderPayGap.WebUI/Views/ActionHubOld/Overview.cshtml

```diff
@@ -0,0 +1,102 @@
+                </span>
+            </h2>
+        </div>
+        <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
+            <p class="govuk-body">
+                This guidance is based on the best available evidence currently.
+                It will be regularly updated as the evidence base develops.
+            </p>
+        
+            <p class="govuk-body">
+                Where possible, we have used evidence based on randomised controlled trials that were conducted in the field and that measure objective outcomes related to recruitment, progression and promotion.
+                High quality evidence is currently scarce in the field of gender equality in the workplace.
+            </p>
+        </div>
+    </div>
+    <div class="govuk-accordion__section ">
+        <div class="govuk-accordion__section-header">
+            <h2 class="govuk-accordion__section-heading">
+                <span class="govuk-accordion__section-button" id="accordion-default-heading-2">
+                    How you can help
+                </span>
+            </h2>
+        </div>
+        <div id="accordion-default-content-2" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-2">
+            <p class="govuk-body">
+                To build better evidence, we encourage researchers and employers to evaluate the actions they take to improve gender equality in the workplace.
+            </p>
+      
+            <p class="govuk-body">
+                The Government Equalities Office will be working with employers to build more evidence on what works.
+                If you represent an organisation with 4000 or more employees in the UK and would like to partner with us, please contact the Gender and Behavioural Insights programme <a class="govuk-link" href="mailto:gabiprogramme@bi.team">gabiprogramme@bi.team</a>
+                <br/>
+                This is a programme run by the Behavioural Insights Team for the Government Equalities Office.
+            </p>
+        </div>
+    </div>
+</div>
+
+@{
+    var rightLink = new PaginationLinkViewModel
+    {
+        Title = "Next",
+        Text = "Effective actions",
+        Url = Url.Action("Effective", "ActionHub")
+    };
+    
+    await Html.RenderPartialAsync("/Views/Components/PaginationLinks/PaginationLinks.cshtml", new PaginationLinksViewModel
+    {
+        LeftLink = null,
+        RightLink = rightLink
+    });
+}
+
+<p class="govuk-body-s govuk-!-margin-top-5">
+    <a class="govuk-link print-link" href="/public/assets/pdf/Evidence-based_actions_for_employers.pdf">
+        Print entire guide (PDF)
+    </a>
+</p>
+
+<div class="pdf-preview">
+    <span class="pdf-preview__image">
+        <a href="/public/assets/pdf/Evidence-based_actions_for_employers.pdf">
+            <img style="border:6px solid #dee0e2" width="78" src="/img/what-works.png" alt=""/>

```

Comment:
I think this image is just for decoration (since the meaning is in the `span` below), so perhaps `alt=""` would be corect
https://en.wikipedia.org/wiki/Alt_attribute#Decorative_images

I suppose really this `<a>` and the next `<a>` should be merged so there's just one link which contains the next and a decorative image

Reply:
I'm not quite sure which approach to take on this one - nesting the image in the text `<a>` seems weird to me (although that's just a gut reaction which could quite plausibly be entirely wrong)...
@Binney any thoughts?

Reply:
I tried doing this so the image and text sit inside the same link and I _really_ don't think it's the right approach - it relies on the `<p>` tag containing the download details (PDF, 8.45MB, 12 pages) overlapping with the link area but _not_ being part of the link. It's also really unclear accessibility-wise that the link image and text and the download details are related when the image and text sit in a link together and the details are separately below...not sure what the best method is here, but I think what we've got here with `alt=""` is actually quite good for accessibility compared to the alternatives.